README.md

```diff
@@ -1,2 +1,234 @@
+
+In the `.env` file you just created change the `NEAR_CREDENTIALS_STORE_PATH` to the root of the `near-credentials` folder. (For example `/home/myname/.near-credentials/`).
+
+Near does not require a new contract deployment for each pair. Each pair is generated automaticly when you push a new pair. See [Contract addresses for NEAR](#contract-addresses) 
+Near is also the only one to support batching of transactions, making it cheaper for you to push data on chain. Please see [Batching](#batching)  
+
+# EVM Setup
+
+Change in the `config.json` the `chainId` and `rpc` to the desired EVM chain. Currently it's configured to use the Aurora EVM chain. 
+
+Change in the `.env` the `EVM_PRIVATE_KEY` to your private key (Not a mnemonic but the key that starts with 0x)
+
+
+### Running
+
+To run:
+
+```bash
+npm run start
+```
+
+## Configuring `config.json`
+
+|Key|Type|Description|
+|---|---|---|
+|networks|Network[]|An array of network configuration. (Explained below)
+|pairs|Pair[]|An array of pricing pairs with there sources (Explained below)
+
+### Network
+
+Configuration for a specific network. Currently two types are supported. `evm` and `near`. You can use each network type multiple types to combine for example Avalanche, Polygon and Ethereum.
+
+### options for every network
+
+These options are available/required for every network
+
+|Key|Type|Description|
+|---|---|---|
+|type|string|Lets the node know what the network type is. See docs below on which types are available.
+|networkId|number|A custom ID that you fill in. This will be used to connect pairs to a specific network configuration. Can be anything you want to identify the configuration|
+|rpc|string|The URL to the network's RPC|
+|wssRpc|string/undefined|The URL to the websocket RPC, is required by some modules (not for price pushing)|
+|blockFetchingInterval|number/undefined|Interval in ms on when to fetch the latest block, default is 5000ms|
+
+### evm
+
+|Key|Type|Description|
+|---|---|---|
+|type|"evm"|Lets the node know this is an EVM type chain|
+|privateKeyEnvKey|string|The name of the env variable where the private key is stored. (can be set in the `.env` file)|
+|chainId|number|The chain id of the EVM chain. (1 = Ethereum)|
+
+Example:
+
+```JSON
+{
+    "networks": [
+        {
+            "type": "evm",
+            "networkId": 133,
+            "privateKeyEnvKey": "EVM_PRIVATE_KEY",
+            "chainId": 1313161554,
+            "rpc": "https://testnet.aurora.dev"
+        }
+    ]
+}
+```
+
+### near
+
+#### accessing / generating NEAR private keys
+There's multiple ways to go about this. The simplest method would be to create, or sign in to, a NEAR account using the [near web wallet](https://wallet.near.org). And then calling `NEAR_ENV={NETWORK} near login` and following the steps provided by the CLI. This will generate a access keys in `~/.near-credentials/{NETWORK}/{MY_ACCOUNT}.near.json` which can then be copied into any environment.
```

Comment:
```suggestion
There's multiple ways to go about this. The simplest method would be to create, or sign in to, a NEAR account using the [near web wallet](https://wallet.near.org). And then calling `NEAR_ENV={NETWORK} near login` and following the steps provided by the CLI. This will generate access keys in `~/.near-credentials/{NETWORK}/{MY_ACCOUNT}.near.json` which can then be copied into any environment.
```